src/components/Form.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react"
+import '../styles/_Form.scss'
```

Comment:
I think you do not need to import the scss file. Would love a confirmation from @rachelsoae